### DISCUSSION TITLE
Strech thumbnails to dial size

### DISCUSSION THREAD

**Discussion by Shinkoku** [fastaddons/GroupSpeedDial#357] [General]
Good day,
Is it possible to stretch thumbnail (taken from the "public library") to dial's size (I use dynamic dials with size set up manually)? 
I tried all the settings I could find (dial settings/general thumbnail settings), but those public thumbnails are always small and fill only a part of the dial. Would like to stretch them, to fill entire dial or at least fit to height or width. That's be marvelous
Thanks for great add on!

**Comment by Juraj-Masiar** [comment]
Hello,
I'm sorry about 2 years delay...
It's most likely caused by the Thumbnail padding - see the "Visual" tab (in the Edit dial window):

<img width="712" height="544" alt="image" src="https://github.com/user-attachments/assets/c2447de3-fe1d-4b15-8027-bc6171c383cd" />
